Navigation/nav.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation | Page</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel="stylesheet" href="../Navigation/nav.css">
+    <script src="https://kit.fontawesome.com/c9ffd32afb.js" crossorigin="anonymous"> </script>
+
+</head>
+<body>
+    
+    <div class="container">
+        <nav>
+          <div class="nav-section left-nav-links">
+            <div>
+              <a class="nav-a-link" href="../index.html" > Geeta UI</a>
+             </div>
+          </div>
+         
+          <div class="nav-section right-nav-links">
+  
+              <ul> 
+                  <li class="nav-li-links">
+                    <span >  <a class="nav-a-link" href="../index.html" >Home</a> </span>
+                  </li>
+       
+                   <li  class="nav-li-links">
+                    <span > <a class="nav-a-link" href="../Docs/docs.html"> Docs</a>   </span>
+                   </li>
+       
+       
+                   <li  class="nav-li-links">   <span >  <button class="theme-button" id="theme-btn" > 
+                      <i class='fas fa-moon'> </i>   </button>
+                   </li>
+               </ul>
+         
+       </div>
+      </nav>
+
+
+
+    <br> <br>
+
+
+      <div class="parent-container">
+
+    
+        <div class="slidebar">
+
+          <ul class="sub-topic-list" > 
+                <h2 >Components</h2>
+        
+                <li class="list-links"> <a href="../Avatar/avatar.html" >   Avatar</li> </a>
+                <li class="list-links"> <a href="../Alert/alert.html" >  Alert</li> </a>
+                <li class="list-links"> <a href="../Badge/badge.html" >  Badge</li> </a>
+                <li class="list-links"> <a href="../Button/button.html" >  Button</li> </a>
+                <li class="list-links"> <a href="../Cards/cards.html" >  Cards</li> </a>
+                <li class="list-links"> <a href="../Image/image.html" >  Image</li> </a>
+                <li class="list-links"> <a href="../Input/input.html" >  Input</li> </a>
+                <li class="list-links"> <a href="../Lists/lists.html" >  List</li> </a>
+                <li class="list-links"> <a href="../Rating/rating.html" > Rating</li> </a>
+                <li class="list-links"> <a href="../Toast/toast.html" > Toast</li> </a>
+                <li class="list-links"> <a href="../Typography/typography.html" >  Typography</li> </a>
+                <li class="list-links"> <a href="../Navigation/nav.html" >  Navigation</li> </a>
+            </ul>
+          </div>
+      
+      
+        <div class="doc-content">
+         <div class="content-title" > Navigation</div>
+         
+         <br> <br>
```

Comment:
This is not a good practice (and also br is an empty tag i.e. it should be like &lt;br /&gt;), instead, you can use margin here.